gensim/test/test_coherencemodel.py

```diff
@@ -138,6 +142,12 @@ def testCvMalletModel(self):
             return
         CoherenceModel(model=self.malletmodel, texts=self.texts, coherence='c_v')
 
+    def testCw2vMalletModel(self):
+        """Perform sanity check to see if c_w2v coherence works with LDA Mallet gensim wrapper"""
+        if not self.mallet_path:
```

Comment:
It's not correct checking, because if you run test & have no `mallet_path`, the status of this test is `ok`, but should be `skipped`